content/en/monitors/guide/template-variable-evaluation.md

```diff
@@ -0,0 +1,98 @@
+| atan(var)           | Returns the arc tangent of var, in radians                                                                                                 |
+| atan2(var1, var2)   | Returns atan(var1 / var2), in radians                                                                                                      |
+| atanh(var)          | Returns the inverse hyperbolic tangent of var                                                                                              |
+| exp(var)            | Returns e (the base of the natural logarithm) raised to the power of var                                                                   |
+| log10(var)          | Returns the base-10 logarithm of var                                                                                                       |
+| abs(var)            | Returns the absolute value of var                                                                                                          |
+| int(var)            | Returns var as an integer                                                                                                                  |
+| float(var)          | Returns var as a float                                                                                                                     |
+| trunc(var)          | Returns var as an integer                                                                                                                  |
+| round(var)          | Returns var rounded to the nearest integer                                                                                                 |
+| ceil(var)           | Returns the ceiling of var (the smallest integer that is greater than or equal to var)                                                     |
+| floor(var)          | Returns the floor of var (the largest integer that is less than or equal to var)                                                           |
+| dec(var)            | Returns the numbers to the right of the decimal point, for example 12.345 returns 0.345.                                                   |
+| sgn(var)            | Returns the value of the sign function evaluated at var:<br>sgn(var) = 1 if var > 0<br>sgn(var) = 0 if var = 0<br>sgn(var) = -1 if var < 0 |
+| humanize_bytes(var) | Returns a human readable formatting of var bytes                                                                                           |
+| humanize_bits(var)  | Returns a human readable formatting of var bits                                                                                            |
+| to_bool(var)        | Returns true if var = 1<br>Returns false if var = 0                                                                                        |
+
+### Examples
+
+Use the int function to evaluate the `{{value}}` of a monitor to improve readability and remove the decimals with the following:
```

Comment:
**Change this sentence to:** If the the decimal places of the `{{value}}` template variable are unimportant for your particular use cases, use the int function to evaluate `{{value}}` as an integer to improve readability and remove the decimals like so: